component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java

```diff
@@ -192,6 +192,7 @@ public static ProfileEntity buildEntityProfile(Profile profile, String restPath,
       User user = organizationService.getUserHandler().findUserByName(userEntity.getUsername());
       if (user != null) {
         userEntity.setIsInternal(user.isInternalStore());
+        userEntity.setCreatedDate(String.valueOf(user.getCreatedDate().getTime()));
```

Comment:
the date we have to display as tooltip is the date of last sync for the user, not the created date.
Imagine that the user is modified in AD, then synchronised, the sync date is not the same as the created date.
As explain in the tech spec, we should add a listener on event linked to synchronisation, and store/update the sync date in the user profile